src/xgov-dapp/src/features/rounds/index.tsx

```diff
@@ -102,10 +74,10 @@ const VotingRounds = () => {
         </Button>
       )}
 
-      {error && (
+      {isError && (
         <Alert className="max-w-xl mt-4 text-white bg-red-600 font-semibold" icon={false}>
           <Typography>Could not load voting rounds:</Typography>
-          <Typography>{error}</Typography>
+          <Typography>{(error as Error).message}</Typography>
```

Comment:
are we sure we want to render the error message instead of logging it?

Reply:
This preserves the previous behavior